texar/data/data/multi_aligned_data.py

```diff
@@ -377,6 +379,11 @@ def default_hparams():
         hparams["datasets"] = []
         return hparams
 
+    def to(self, device: torch.device):
```

Comment:
`to` is defined in `DataBase` which simply sets `self.device`. `MultiAlignedData` needs a different implementation because movement of data is not explicitly done in its `collate` method, but is rather done by the `collate` methods of the constituting databases. So here we need to call `.to` on those databases.

Reply:
Ah I see. Makes sense.